public/app/features/commandPalette/KBarResults.tsx

```diff
@@ -122,7 +123,10 @@ export const KBarResults = (props: KBarResultsProps) => {
 
       if (item.command) {
         item.command.perform(item);
-        query.toggle();
+        // TODO: ideally the perform method would return some marker or we would have something like preventDefault()
```

Comment:
Hacky way to prevent closing cmdK when selecting scopes as you can select multiple ones. Would like to figure it out in separate PR though.